### ISSUE TITLE
Trace Comment on 1.4.11  Resize Content 400%  (Level A)

### ISSUE THREAD

**Issue by GreggVan** [w3c/wcag21#281]
MY COMMENTS:

I love this SC but 

1.	I don't think this should be at  Level A 
We try to reserve level A to things that cannot be made accessible by any other means if you don't do this.   

2.	This is an example of an SC that we cannot follow ourselves with our own tools.  So it should not be a Level A.  And it should not be a Level AA either unless and until    
     a) we meet this with all of our web pages and web tools.
OR
     b) for those pages/tools we have no control over - we show that we CAN meet the SC by creating alternate versions of the pages / tool pages  that show how it could/should be done.  

SUGGESTED RESOLUTION:
Move it to Level AAA until we prove it is doable enough (or change it so that it is doable enough) that we meet the SC ourselves.

(PS  EDITORIAL SUGGESTION)  The wording at the end of the SC is hard to read because     " necessary to use"   sounds like it should be read like it would in the sentence  "it is necessary to use"      
SUGGESTED EDITORIAL RESOLUTION:  change the ending to  " to use, or to preserve meaning of the content."

**Comment by alastc** [comment]
(Commenting as myself, not the working group.)

> We try to reserve level A to things that cannot be made accessible by any other means if you don't do this.

I'll refer to [Wayne's comment](https://github.com/w3c/wcag21/issues/250#issuecomment-293123655) on this, if something requires 40-100 times as much effort to read, does that not qualify?

> This is an example of an SC that we cannot follow ourselves with our own tools. So it should not be a Level A. And it should not be a Level AA either unless and until
...
> b) for those pages/tools we have no control over - we show that we CAN meet the SC by creating alternate versions of the pages / tool pages that show how it could/should be done.

Sure, if you switch your user-agent string to a mobile agent, this page meets the SC:

<img alt="screenshot of this page in mobile view at 400%" src="https://alastairc.ac/tmp/github-mobile.png">

Is that easy to do as a user? Not yet, but it is certainly possible to do as a developer. Sites like [the BBC started the process](http://www.bbc.co.uk/news/technology-31966686) of going responsive by creating a mobile-oriented version, then expanding that to cater to all sizes of screen. Perhaps Github is doing that?

More relevant to whether this is doable, I've just checked the default home pages of: BBC.co.uk, Adobe.com, microsoft.com, theguardian.com, all good at 400%. These are high-traffic, high profile sites. And my blog is fine to, for a non-high profile / high-traffic site I put together in two evenings.

Copying from my comment to Webaim: I've been over this with a few clients recently and the principle is not scary, they were already well on their way to catering to 400%. The details were about catching things like sticky headers which assume a portrait view, whereas non-mobiles are landscape, so the sticky header took up 1/3 of the view. (Vertical media queries to un-stick the header solve that.)

The [umd.edu site](https://www.umd.edu/) (if you're familiar with that?) is a good example of this, it is generally fine at 400% except that the sticky header assumes a portrait view. If that were unstuck the homepage would be fine. The [Trace homepage](https://trace.umd.edu/) is ok, but just needs to let the columns reflow (using media queries like the main site does).

**Comment by GreggVan** [comment]
> On Apr 24, 2017, at 7:58 PM, Alastair Campbell <notifications@github.com> wrote:
> 
> (Commenting as myself, not the working group.)
> 
> We try to reserve level A to things that cannot be made accessible by any other means if you don't do this.
> 
> I'll refer to Wayne's comment <https://github.com/w3c/wcag21/issues/250#issuecomment-293123655> on this, if something requires 40-100 times as much effort to read, does that not qualify?
> 
> 
Certainly.  If there is no way that AT can be used to solve the problem.   If I could use AT however to re-present the content so I could read it (or operate it) reasonably - then only the provisions needed to make it work with AT would be level A.   


> This is an example of an SC that we cannot follow ourselves with our own tools. So it should not be a Level A. And it should not be a Level AA either unless and until
> ...
> b) for those pages/tools we have no control over - we show that we CAN meet the SC by creating alternate versions of the pages / tool pages that show how it could/should be done.
> 
> Sure, if you switch your user-agent string to a mobile agent, this page meets the SC:
> 
> 
This page does. But can we do it for any page we find on the internet.  Not that they meet the SC now — but we need to be able to show that ANY page CAN meet the SC  - and we can show exactly how it can — before we REQUIRE that all pages meet the SC for conformance. 

>  <https://camo.githubusercontent.com/f44f9bb6532452341f0527cb5d54af4a3da0c8a3/68747470733a2f2f616c617374616972632e61632f746d702f6769746875622d6d6f62696c652e706e67>
> Is that easy to do as a user? Not yet, but it is certainly possible to do as a developer. Sites like the BBC started the process <http://www.bbc.co.uk/news/technology-31966686> of going responsive by creating a mobile-oriented version, then expanding that to cater to all sizes of screen. Perhaps Github is doing that?
> 
> 
Again - the question isn’t can we do it on some pages — but are we sure it can be done on any page.   Could we design any page to meet it. 
> More relevant to whether this is doable, I've just checked the default home pages of: BBC.co.uk, Adobe.com, microsoft.com, theguardian.com, all good at 400%. These are high-traffic, high profile sites. And my blog is fine to, for a non-high profile / high-traffic site I put together in two evenings.
> 
> 
Great.   The question is - can it apply to all sites.   I am not arguing that it can’t.  Just saying that we need to convince ourselves  that it can.  


Are you talking about 400% zoom?  or do you mean 400% larger text with reflow?

**Comment by alastc** [comment]
> If I could use AT however to re-present the content so I could read it (or operate it) reasonably - then only the provisions needed to make it work with AT would be level A.

Without the author enabling reflow, that is very difficult. The reason 'linearisation' was separated is that it is a sledgehammer approach - killing the site's layout. Even the 'reader' views (e.g. in Safari which pull out the main text are limited because you then can't navigate.

> This page does. But can we do it for any page we find on the internet.  Not that they meet the SC now — but we need to be able to show that ANY page CAN meet the SC  - and we can show exactly how it can — before we REQUIRE that all pages meet the SC for conformance.

Yes. Find me a site that doesn't want to work on mobile!

The exception for 2D-ness is fair broad, and a good reason to keep the 1.4.4 200% with scrolling SC as well. 

The only 'tricky' interface for this SC is IDEs (editing interfaces) where you need to have a content area and a toolbar and possibly another block that all need to be on screen at the same time. I'd like to say that is covered by the 2D exception in the understanding document, does that seem reasonable?

> Are you talking about 400% zoom?  or do you mean 400% larger text with reflow?

Sorry, you'll have to help me with the distinction here?

400% zoom is the general mechanism in browsers, we're _**not**_ talking about text-resize. The whole layout can reflow whilst zooming when media-queries are used (as specified by the author). 

You can achieve the same thing when the user resizes text as well by sizing everything (relevant) in EMs, but that is more difficult and not required.

**Comment by DavidMacDonald** [comment]
# Test heading

**Comment by NicoleAcc76** [comment]
alstc: Do I understand it correctly that, reading your explanation, most web applications fall under the 2D exception? Wouldn't it be better to put that into the requirement itself than increasing the cognitive workload by spreading information to many sources?

**Comment by alastc** [comment]
@NicoleAcc76 No, it would be where elements in 2 dimensions have to be on screen at the same time, e.g. an editing toolbar plus slide thumbnails. 

That doesn't affect that many applications, for example, this Github interface doesn't try to display columns next to this edit box, just a small toolbar above the edit box.

Also, how would we define a web application in such as way we could include it in the SC text? Anything with a form element could be defined as a web application.

**Comment by GreggVan** [comment]
> On Apr 25, 2017, at 4:50 AM, Alastair Campbell <notifications@github.com> wrote:
> 
> If I could use AT however to re-present the content so I could read it (or operate it) reasonably - then only the provisions needed to make it work with AT would be level A.
> 
> Without the author enabling reflow, that is very difficult.
> 
????   these AT take the text and re-present it as flowing text.   as long as the text is programmatically determinable — they can re present it in any font size face style etc.  

> The reason 'linearisation' was separated is that it is a sledgehammer approach - killing the site's layout. Even the 'reader' views (e.g. in Safari which pull out the main text are limited because you then can't navigate.
> 
Yes it is a more powerful approach but I’m not sure I would use the word sledgehammer.  if so - then all screen readers are sledgehammer approaches as well.        


Don’t get me wrong.   I love what we are trying to do here.   I just want to be sure we do it right.


> This page does. But can we do it for any page we find on the internet. Not that they meet the SC now — but we need to be able to show that ANY page CAN meet the SC - and we can show exactly how it can — before we REQUIRE that all pages meet the SC for conformance.
> 
> Yes. Find me a site that doesn't want to work on mobile!
> 
I'm not sure that is the question.   I think the question is ‘are there people who cannot figure out how to make their sites work on mobile and still meet the proposed SC.  
> The exception for 2D-ness is fair broad, and a good reason to keep the 1.4.4 200% with scrolling SC as well.
> 
> 
agree

> The only 'tricky' interface for this SC is IDEs (editing interfaces) where you need to have a content area and a toolbar and possibly another block that all need to be on screen at the same time. I'd like to say that is covered by the 2D exception in the understanding document, does that seem reasonable?
> 
> Are you talking about 400% zoom? or do you mean 400% larger text with reflow?
> 
> Sorry, you'll have to help me with the distinction here?
> 
Pinch zoom on a phone vs   larger text and objects with reflow.

**Comment by NicoleAcc76** [comment]
Thanks Alastair,
It is just that I have with some requirements the feeling, they work quite well for a simple web site with lots of text content, but when we talk about applications it is getting harder to impossible. It is not only about technical possibility but also to meet all other requirements in parallel.
200% works quite well, but taking dependencies due to e.g. display sizes into consideration 400% is getting difficult. Also regarding the understandability of the requirements, wouldn’t it make sense to make two of them – one with the 2D exception and the 400% extra?

**Comment by alastc** [comment]
@GreggVan wrote:
> ????   these AT take the text and re-present it as flowing text.   as long as the text is programmatically determinable — they can re present it in any font size face style etc.

Yes, AT can do that. However, this SC is intended to be within the bounds of what authors define, working with browser zoom. The AT approach of _overriding layout_ it is a very blunt instrument, and you get a different set of content issues. The moment you override layout, it becomes a very different content requirement.

The user-need behind Resize content and Linearise are the same (apart from a matter of degree), but the content requirements are quite different. 

> I think the question is ‘are there people who cannot figure out how to make their sites work on mobile and still meet the proposed SC.

I think the only way we're going to find out is putting it out there. Mobile traffic is now more than desktop traffic, any client we've had who isn't using a responsive and/or mobile specific site is already working on it. Taking into account the exceptions already in the SC, I can't think of any.

> > > Are you talking about 400% zoom? or do you mean 400% larger text with reflow?

> > Sorry, you'll have to help me with the distinction here?

> Pinch zoom on a phone vs   larger text and objects with reflow.

Pinch zoom on a mobile works in a different way, it does not trigger reflow, and there's nothing the author can do about that. We'll need to note it in the understanding, but we removed the SC's exception for that layout/zoom style because it isn't a content requirement. 

I'm still not sure how that related to the 2D exception, or what you were asking there?

@NicoleAcc76 wrote:
> It is just that I have with some requirements the feeling, they work quite well for a simple web site with lots of text content, but when we talk about applications it is getting harder to impossible. 

Do you have any examples of where it wouldn't be possible? Most sites and apps are designed to work on mobile these days, which allows for the same mechanism to work for desktop zoom. The only instances I'm aware of are editing-style interfaces where multiple toolbars / content areas need to be onscreen at the same time. 
It is notable that certain apps (e.g. Powerpoint / Google presentation, excel/Sheets) do not have responsive web versions and the mobile interfaces take a completely different approach.

The intent it to cover these with the "2D exception", but do you have other examples in mind?


> Also regarding the understandability of the requirements, wouldn’t it make sense to make two of them – one with the 2D exception and the 400% extra?

We effectively do, the current 1.4.4 (200% and allowing horizontal scrolling) would still be in effect.

**Comment by NicoleAcc76** [comment]
Hi alastc,

Had a meeting with my accessibility team mates. 
Examples where we see difficulties by 400%:
Tile layout 
Tree Layout with a deep hierarchy
Most forms, as instruction and fields cannot be seen at the same time
Most applications with a boxed layout for several tasks

We still have several concerns with this SC and its wording.

First, it is unclear form the wording what exactly is the exception:
Are “parts of the content where fixed spatial layout is necessary to use or meaning.” excepted from being zoomed or excepted from the need to not scroll in two dimensions?

Next, the cognitive workload to understand what is the aim to avoid two dimensional scrolling is high. In addition, it opens backdoors.
From further comments we understand the following is meant:
Horizontal scrolling is to be avoided for languages written horizontally, and vertical scrolling is to be avoided for languages written vertically.
To sum it up, scrolling in reading direction is to be avoided, while scrolling perpendicularly to it is allowed.
This should be explained somewhere.

Next, we still agree that SC 1.4.4 and this SC should fit nicely together and receive comprehensible levels.
It is not comprehensible to have a level A for 400% and level AA for 200%.

Further we noticed by recalling the discussion and the testing instructions, that 400% is meant to be fulfilled on a desktop monitor and on a laptop, but not on a mobile device. This should be reflected in the SC itself. 

ISO Standards also deal with the "Minimum Fonts Size" has that ever been a topic?

Best
Nicole

**Comment by alastc** [comment]
Hi @NicoleAcc76,

For the examples, there are techniques for those, although I'm not sure what the different between tiles and boxes is?

Brad Frost has [collected many](https://bradfrost.github.io/this-is-responsive/patterns.html), including [misc box sizes](http://codepen.io/bradfrost/full/Dybsz), [4 columns reducing down](http://codepen.io/bradfrost/full/LuAet), and [form fields that change layout](http://codepen.io/chriscoyier/full/DmnlJ) as you zoom in.

A tree should work, I note [this tree example](http://www.oracle.com/webfolder/technetwork/jet/jetCookbook.html?component=tree&demo=basicTree) works at 400% although it isn't massively deep, I think that would work to at least 6 levels deep.

> Are “parts of the content where fixed spatial layout is necessary to use or meaning.” excepted from being zoomed or excepted from the need to not scroll in two dimensions?

As the zoom is under user control, we can't except something from zoom, only the effect that the zoom has, i.e. scrolling.

> To sum it up, scrolling in reading direction is to be avoided, while scrolling perpendicularly to it is allowed.

Sort of, but really it means: Scrolling in the default direction is fine (and inevitable), but scrolling in two directions is to be avoided.

It will be explained, but if you have any suggestions for something instead of the current SC text I'm very open to that!

> It is not comprehensible to have a level A for 400% and level AA for 200%.

I agree, I'd like to swap the levels, but the balance is against changing SCs from WCAG 2.0, for which there is resistance.
Would you mind updating the level of an SC from WCAG 2.0?

> 400% is meant to be fulfilled on a desktop monitor and on a laptop, but not on a mobile device. This should be reflected in the SC itself.

The SC originally did have an exception, but it was removed because it is not a requirement of the content. It is the content (via CSS & media queries) that has to be able to adapt. To make testing feasible, that cannot include testing across all devices, so the requirements need to be of the content, of what an author can adjust.
However, we will acknowledge what happens on mobile devices on the understanding page.

> ISO Standards also deal with the "Minimum Fonts Size" has that ever been a topic?

Indirectly it is discussed a lot, but problem is that sizing on the web is relative, you can't predict what physical size it will be. The recent versions of that discussion have been about using CSS pixels (for which there is a definition now) and it's relation to EM units and physical sizing.

**Comment by GreggVan** [comment]
> On May 5, 2017, at 5:26 AM, Alastair Campbell <notifications@github.com> wrote:
> 
> @GreggVan <https://github.com/GreggVan> wrote:
> 
> ???? these AT take the text and re-present it as flowing text. as long as the text is programmatically determinable — they can re present it in any font size face style etc.
> 
> Yes, AT can do that. However, this SC is intended to be within the bounds of what authors define, working with browser zoom. The AT approach of overriding layout it is a very blunt instrument, and you get a different set of content issues. The moment you override layout, it becomes a very different content requirement.
> 
> 

That would then mean that Screen readers are then are all blunt instruments.  — and the WCAG has limited almost all accessibility provisions to for blind users  to blunt instrument access. 

I’m not sure I agree with this assessment.

I like that content can be accessible without AT.  I think everyone would. 

But in the past we haven’t put things at level A that have other solutions including AT. 

(and I really don’t think that AT should be relegated to the category of blunt instruments.)

best

g
> user-need behind Resize content and Linearise are the same (apart from a matter of degree), but the content requirements are quite different.
> 
> I think the question is ‘are there people who cannot figure out how to make their sites work on mobile and still meet the proposed SC.
> 
> I think the only way we're going to find out is putting it out there. Mobile traffic is now more than desktop traffic, any client we've had who isn't using a responsive and/or mobile specific site is already working on it. Taking into account the exceptions already in the SC, I can't think of any.
> 
> Are you talking about 400% zoom? or do you mean 400% larger text with reflow?
> 
> Sorry, you'll have to help me with the distinction here?
> 
> Pinch zoom on a phone vs larger text and objects with reflow.
> 
> Pinch zoom on a mobile works in a different way, it does not trigger reflow, and there's nothing the author can do about that. We'll need to note it in the understanding, but we removed the SC's exception for that layout/zoom style because it isn't a content requirement.
> 
> I'm still not sure how that related to the 2D exception, or what you were asking there?
>

**Comment by alastc** [comment]
I wasn't trying to make a comparison to screen readers, I was comparing two methods of adjusting zoom/layout.

For example, this is Microsoft.com at 400%, what we expect from the Resize SC:

## Resize content to 400% (via browser zoom)
<img src="https://alastairc.ac/tmp/ms-400.png" alt="Microsoft homepage, mobile style appearance">

<hr>

And this is the same page applying a linearisation tool:

## Linearise content, also at 400% (via a bookmarklet)
<img src="https://alastairc.ac/tmp/ms-linear-400.png" alt="Microsoft homepage, reformatted and a bit all over the place">

I'm confident that the resize content SC is feasible, useful, and fits WCAG nicely.

The linearise SC is also useful (for going more than 400%), probably feasible (to be proven), and should be made to fit WCAG. But I'm less confident.

**Comment by GreggVan** [comment]
ok….

but  not sure what you are trying to say / illustrate …

can you elaborate

G


PS  linearization bookmarklets are not very great AT

**Comment by alastc** [comment]
> but  not sure what you are trying to say / illustrate …

That a method of zoom which works according to how the author has specified the layout is a more elegant & robust approach than the user completely overriding the layout. 

That's where the 'blunt instrument' comment came from, because overriding the layout is like taking a sledgehammer to a sun lounger instead of adjusting the position of the seat-back. It will be flat in both cases, but you have less control over where the bits end up when using the hammer.

> PS  linearization bookmarklets are not very great AT

Perhaps not, but that type of mechanism was a very strong requirement coming from the LVTF, and having that as an SC reduced the need for a larger than 400% increase under author control.

Regarding the A/AA levels, we have three SCs on the table about making things bigger:

1. Resize text 1.4.4 at 200% increase, allowing scrolling, no exceptions (the current 2.0 SC), level AA.
2. Zoom content, at 400%, no scrolling, exception for essential 2D layouts.
3. Reflow to single column (linearisation), allow for > 400% by overriding layout, no scrolling, exception for essential 2D layouts.

Reflow is the only one that needs "AT" (if browser overrides qualify as AT), and it will be the newest / oddest criteria for developers.

My preference (and several comments indicated that it) would be best to adjust the current SC to level A so we have:

* A Resize text (applies to all text)
* AA Zoom content (applies to all content except things than need to be 2D)
* AA Reflow to single column (applies to all content except things than need to be 2D)

Does that make sense for WCAG leveling?

**Comment by GreggVan** [comment]
> A Resize text (applies to all text)
> AA Zoom content (applies to all content except things than need to be 2D)
> AA Reflow to single column (applies to all content except things than need to be 2D)
> Does that make sense for WCAG leveling?
> 
We did not try to find something for each level.    The levels had their own meaning — as described in Understanding WCAG 2.0. 

But generally for  level A  - it usually means that if you don’t do this there is NO WAY for the user to do this through any other means — including AT.    There were a couple of exceptions but that was one guiding principle. 

Level AA  was for everything else important, testable, generally applicable to all content

Level AAA was for things important, testable, but not generally applicable (could not be applied in all cases.  This was for people wanting to go the extra mile

There were of course many other SC we tried to create but could not meet the testable criteria so we made them advisory since we couldnt require something we couldn’t test

**Comment by GreggVan** [comment]
> A Resize text (applies to all text)
> AA Zoom content (applies to all content except things than need to be 2D)
> AA Reflow to single column (applies to all content except things than need to be 2D)
> Does that make sense for WCAG leveling?
> 
We did not try to find something fro each topic for each level.    The levels had their own meaning — as described in Understanding WCAG 2.0. 

But generally for  level A  - one criteria was  “if the author doesn't this, there is NO WAY for the user to do this through any other means — including AT. There were a couple of exceptions but that was one guiding principle. 

Level AA  was for everything else important, testable, generally applicable to all content

Level AAA was for things important, testable, but not generally applicable (could not be applied in all cases.) In one case, it was because there were not enough sign language interpreters in existence to provide sign language interpretation for all web pages.   This was for people wanting to go the extra mile.    

There were of course many other topics we tried to create SC fro, but could not meet the testable criteria so we made them advisory (since we couldnt require something we couldn’t test -(so a person could not tell if they had passed).   


and in answer to your question — I can’t tell from the quick characterizations you did.

Your  level A looks like it requires even 2D layouts to be reflowed 
You don’t distinguish between what the author needs to do, what authors must build into their UI,  and what must be ‘possible’ by AT or access features in user agents. 


> On May 10, 2017, at 4:30 AM, Alastair Campbell <notifications@github.com> wrote:
> 
> but not sure what you are trying to say / illustrate …
> 
> That a method of zoom which works according to how the author has specified the layout is a more elegant & robust approach than the user completely overriding the layout.
> 
> That's where the 'blunt instrument' comment came from, because overriding the layout is like taking a sledgehammer to a sun lounger instead of adjusting the position of the seat-back. It will be flat in both cases, but you have less control over where the bits end up when using the hammer.
> 
> 
I understand blunt instrument.  but your assumption that all AT was a blunt instrument because you can point to one that is was my concern. 

Why do you assume that AT would have to completely override layout to make text reflow?    

> PS linearization bookmarklets are not very great AT
> 
> Perhaps not, but that type of mechanism was a very strong requirement coming from the LVTF, and having that as an SC reduced the need for a larger than 400% increase under author control.
> 
> 
Linearization bookmarklets are a requirement out of LVTF?    or reflow.

Bookmarklets are not the best form of AT.     

I was just commenting that choosing a very light weight AT and then using it to say all AT was not good enough seemed to be pushing the argument.     

> Regarding the A/AA levels, we have three SCs on the table about making things bigger:
> 
> Resize text 1.4.4 at 200% increase, allowing scrolling, no exceptions (the current 2.0 SC), level AA.
> Zoom content, at 400%, no scrolling, exception for essential 2D layouts.
> Reflow to single column (linearisation), allow for > 400% by overriding layout, no scrolling, exception for essential 2D layouts.
> Reflow is the only one that needs "AT" (if browser overrides qualify as AT), and it will be the newest / oddest criteria for developers. 
> 
> 

#2 needs reflow too doesn’t it? 

or do all pages have to be in 4 column form? 

Not sure I understand how #2 would not require text to be reflowed to a narrower window.


thx

gregg

**Comment by alastc** [comment]
> Why do you assume that AT would have to completely override layout to make text reflow?

I'm not, we have two SCs in this area. This one is reflow within the authors control. The Linearise one (#58) is to allow users to override the layout. That is essentially the demo above.

Anyway, the SC text has gone through significant revision, [latest version here](https://rawgit.com/w3c/wcag21/resize-content_ISSUE-77/guidelines/sc/21/resize-content.html).

I have responded to the [necessity & feasibility aspects](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Resize_content_issues_review#Initial_response:) and it is currently at level AA.

Is there anything you can't live with before it goes into the draft?

**Comment by DavidMacDonald** [comment]
I've accommodated and trained employees with low vision employees in banks and government for about 15 years. My experience is that once we need more  than 400% (4x zoom), we are really into screen reader territory. I generally say "OK lets start thinking about migrating to learning a screen reader."

Almost all users I've worked at were extremely relieved to transition to screen reader at that point. One employee hung onto zooming til it was 20x. (about 5 letters on a 27"screen) When we finally went to screen reader she was extremely relieved and wish she had done it sooner.

I don't think WCAG 2.1 should try to accommodate more than 400% without assistive technology. I think the ask upon the developer is disproportionate to the benefit the the end user.

I think after 400% the end user either needs AT like ZoomText's docreader app which reflows content, a screen reader, or other other means. I don't think it should be the author's responsibility.

**Comment by alastc** [comment]
Hi David,

Could you add that comment to the #58 issue then? This is for #77 (zoom content).

I think you'll get some push back from the LVTF on that, they were pushing for x11 as a reasonable max for vision (not x20!). You also have the blurry line of AT, where over-riding styles for linearisation is sort of an AT, just a fairly basic one.

**Comment by alastc** [comment]
I suggest closing this as super-ceded by #343, I can, but perhaps @awkawk should?

**Comment by steverep** [comment]
Recommend this be closed as superseded by later issues as noted by @alastc.  Also, the editorial issue no longer applies to the current SC text.

**Comment by joshueoconnor** [comment]
Thanks @steverep and @alastc - closing as super-ceded by #343
